### ISSUE TITLE
createFareMultiplier()

### ISSUE THREAD

**Issue by sophsong** [learn-co-students/js-advanced-functions-first-class-functions-lab-nyc-web-060319#5]
TLDR - Tests and readme do not align,
propose update to tests to change it to be a multiplier of 5

The Readme says: 
+ `createFareMultiplier()` — This is a higher-order function that takes in one argument, an integer, and returns a function that will multiply a fare for a ride accordingly. If `createFareMultiplier()` receives an argument of `4`, it will return a function that takes in a fare as an argument and quadruples the fare.

however the tests: 
  it('should multiply a given value using the created multiplier', function () {
      const fareQuintupler = createFareMultiplier(5);

      expect(fareQuintupler(5)).to.eql(25);

@EvansJWang 
    });

**Comment by TimeSmash** [comment]
I agree with this statement. The readme says nothing about how the fare is multiplied by itself--it should be more explicit that in the inner function, `fare` will be whatever `int` was. The way this is currently worded implies that another argument should be in the inner function.
